pallets/capacity/src/lib.rs

```diff
@@ -78,6 +78,9 @@ mod mock;
 #[cfg(test)]
 mod tests;
 
+#[cfg(test)]
+mod withdraw_unstaked_tests;
```

Comment:
If we want to follow a pattern of separate test files containing unit tests related to each extrinsic and its helper functions, staking / unstaking should probably also follow this pattern

Reply:
I am a fan of more modular pieces, personally. I also like consistency.

Reply:
How about for now we put these in tests and create a ticket to break these up?

Reply:
I was really hoping to start a trend with this one.  The tests file is getting pretty long.

Reply:
So we decided to break these up slightly differently